src/navigator_data_ingest/tests/test_doc_to_pdf_conversion.py

```diff
@@ -0,0 +1,43 @@
+import re
+import pytest
+import fitz
+
+from Levenshtein import distance
+
+from navigator_data_ingest.base.doc_to_pdf_conversion import convert_doc_to_pdf
+
+
+def all_text(doc: fitz.Document) -> str:
+    text = ""
+    for page in doc:
+        text += re.sub("( |\n|\t)+", " ", page.get_text().replace("\n", " "))
+    return text
+
+
+@pytest.mark.skip(reason="This test requires libreoffice in the environment")
+def test_convert_doc_to_pdf():
+    with open(
+        "src/navigator_data_ingest/tests/fixtures/sampe-01-for-conversion.doc", "rb"
```

Comment:
typo (also below, and nn in the filenames). Also, we probably want a more specific filename or a subdir that indicates these files are used for word to pdf testing?

```suggestion
        "src/navigator_data_ingest/tests/fixtures/sample-01-for-conversion.doc", "rb"
```